@xen-orchestra/web-core/lib/locales/en.json

```
@@ -308,6 +328,7 @@
   "new-vm.name": "VM name",
   "news": "News",
   "news-name": "{name} news",
+  "nic-type": "nic type",
```

According to Figma, the translation should be:
```suggestion
  "nic-type": "NIC type",
```